css/index.css

```diff
@@ -259,15 +238,65 @@ a:active {
 }
 
 .grid-container > div {
-  background-color: bisque ;
+  background-color: bisque;
   text-align: center;
   padding: 20px 0;
   font-size: 17px;
 }
-#skills-list{
-    list-style-type: none;
-    padding: 0;
+#skills-list {
+  list-style-type: none;
+  padding: 0;
 }
 #skills-list li {
-    margin:10px;
-}
\ No newline at end of file
+  margin: 10px;
+}
+form{
+  margin: 0 auto;
+  padding: 20px;
+  border-radius: 8px;
+  background-color:#f9f9f9;
+  box-shadow: 0 0 10px rgba(0,0,0,0.1)
+}
+
+form label{
+  display:block;
+  margin-bottom:5 px;
+  font-size: 1 rem;
+  color:#333;
+}
+#messages{
+  margin:20px auto;
+  padding:20px;
+  border-radius:8px;
+  background-color: #f4f4f4;
+  box-shadow: 0 0 10px rgba(0,0,0,0.1);
+}
+#messages h2{
+  font-size:1.5 rem;
+  color:#333;
+  margin-bottom:20px;
+  text-align: center;
+}
+.message-container{
+  overflow-y:auto;
+}
+#message-list{
```

Comment:
Update CSS styling for message-list

.message-list {
    list-style-type: none;
    padding: 0;
    margin: 0;
    display: flex;
    justify-content: space-between;
    align-items: center;
}

.message-list li span {
    display: block;
}

.message-list > .remove-button {
    background-color: #F00;
    color: #fff;
    border: 1px solid #F00;
    padding: 5px 10px;
    border-radius: 10px;
    cursor: pointer;
    font-size: 0.875;
    margin-left: 0.5rem;
}